### ISSUE TITLE
Prevent zalgo colors bug

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2411]
https://github.com/Marak/colors.js/issues/285

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.28.12](https://github.com/open-wa/wa-automate-nodejs/compare/4.28.11...4.28.12) (2022-01-10)

- 🚑  fix #2411 [`#2411`](https://github.com/open-wa/wa-automate-nodejs/issues/2411)
- Release 4.28.12 [`176e0c0`](https://github.com/open-wa/wa-automate-nodejs/commit/176e0c00b0d86a5726a8dee05c7550b89d586586)
- updated types-only package [`9b808dc`](https://github.com/open-wa/wa-automate-nodejs/commit/9b808dcfb696720ff0a2cd9ab35b9989aa857082)
